src/app-cli-base-coomand.ts

```diff
@@ -0,0 +1,36 @@
+import { resolve } from "path";
+import { existsSync, readFileSync } from "fs";
+
+import config from "./config";
+import { AppManifest } from "./types";
+import { BaseCommand } from "./commands/app/base-command";
+
+export abstract class AppCLIBaseCommand extends BaseCommand<
+  typeof AppCLIBaseCommand
+> {
+  protected manifestPath!: string;
+  protected manifestData!: AppManifest & Record<string, any>;
+
+  /**
+   * The `start` function call getManifestData which reads manifest file is current working directory is app directory
+   */
+  start() {
+    this.getManifestData();
+  }
```

Comment:
Can we convert this start method as init inorder to auto-trigger instead of calling the start method for all the child classes.?